test/test_slurm.py

```diff
@@ -45,6 +45,16 @@ def test_parse_line(self):
             ('297720.batch|batch|||2013-10-25T12:11:20|2013-10-25T12:11:36|00:00:16|16||1|1|wn37|3228K|23820K|COMPLETED'),
             ('321439.batch|batch|||2013-10-27T17:09:35|2013-10-28T04:47:20|11:37:45|41865||1|1|wn16|770728K|1.40G|COMPLETED'),
             ('320816.batch|batch|||2013-10-27T14:56:03|2013-10-28T05:03:50|14:07:47|50867||1|1|wn33|1325232K|2.22G|COMPLETED'),
+            # These lines use TotalCPU rather than CPUTimeRAW.
```

Comment:
So these new test cases make use of the `d-h:m:s` syntax as highlighted in bold?

> ('5940815|cream_877733810|user1|group1|2018-06-03T02:11:12|'
>  '2018-06-05T03:36:08|**2-01:24:56**|**8-15:40:36**|partition|8|1|'
>  'wn1|||COMPLETED'),

Reply:
Kinda. There should be a test for each new format: `d-h:m:s`, `h:m:s` and `m:s.s`

Reply:
ah yes, I found it quite hard to parse the test cases - maybe add a comment stating which new test case tests what format?

Reply:
Done.